swift-sdk/Internal/AnonymousUserManager.swift

```diff
@@ -144,37 +126,32 @@ public class AnonymousUserManager: AnonymousUserManagerProtocol {
     
     // Syncs locally saved data through track APIs
     public func syncEvents() {
-        let events = localStorage.anonymousUserEvents
-        var successfulSyncedData: [Int] = []
-        
-        if let _events = events {
-            for var eventData in _events {
+        if let events = localStorage.anonymousUserEvents {
+            for var eventData in events {
                 if let eventType = eventData[JsonKey.eventType] as? String {
                     eventData.removeValue(forKey: JsonKey.eventType)
                     switch eventType {
                     case EventType.customEvent:
-                        IterableAPI.implementation?.track(eventData[JsonKey.eventName] as? String ?? "", withBody: eventData, onSuccess: {result in
-                            successfulSyncedData.append(eventData[JsonKey.eventTimeStamp] as? Int ?? 0)
-                        })
+                        IterableAPI.implementation?.track(eventData[JsonKey.eventName] as? String ?? "", withBody: eventData)
                         break
                     case EventType.purchase:
                         var total = NSNumber(value: 0)
                         if let _total = NumberFormatter().number(from: eventData[JsonKey.Commerce.total] as! String) {
                             total = _total
                         }
                         
-                        IterableAPI.implementation?.trackPurchase(total, items: convertCommerceItems(from: eventData[JsonKey.Commerce.items] as! [[AnyHashable: Any]]), dataFields: eventData[JsonKey.dataFields] as? [AnyHashable : Any], createdAt: eventData[JsonKey.Body.createdAt] as? Int ?? 0, onSuccess: {result in
-                            successfulSyncedData.append(eventData[JsonKey.eventTimeStamp] as? Int ?? 0)
```

Comment:
What was this callback supposed to do?

Reply:
We determine nothing haha. It is just storing synced data locally.